test/Interpreter/imported_objc_generics.swift

```diff
@@ -105,11 +105,11 @@ ImportedObjCGenerics.test("ProtocolConstraints") {
 }
 
 ImportedObjCGenerics.test("ClassConstraints") {
-  func makeContainedAnimalMakeNoise<T>(_ x: AnimalContainer<T>) -> NSString {
-    return x.object.noise
+  func makeContainedAnimalMakeNoise<T>(x: AnimalContainer<T>) -> NSString {
+    return x.object.noise as NSString
   }
   let petCarrier = AnimalContainer(object: Dog())
-  expectEqual("woof", makeContainedAnimalMakeNoise(petCarrier))
+  expectEqual("woof", makeContainedAnimalMakeNoise(x: petCarrier))
```

Comment:
Um, it's been a couple of weeks... I don't quite remember why this change was necessary. Thanks for catching! I'll double-check to see what's up.